### ISSUE TITLE
polar2canvas returns NaN?

### ISSUE THREAD

**Issue by pstevovski** [wcandillon/react-native-redash#422]
Hi William!

I tried following your tutorials (https://www.youtube.com/watch?v=UMvw20nRZls and https://youtu.be/wEVjaXK4sYQ?t=3371) in particular, to try and create an arc slider. But, when I tried using `polar2canvas` function that redash provides, it throws an error saying: 

`Invariant Violation: [502,{"type":"transform","transform":[{"property":"translateX","value":"<<NaN>>"}]}] is not usable as a native method argument` 

I'm using Expo version 40 and react-native-redash: 16.0.8.

Heres a sample of the code. Please note that the other coordinates function, the `canvas2polar`, works fine:
```ts
interface ArcCursorProps {
    RADIUS: number;
    THETA: Animated.Value<number>
}

const ArcCursor: React.FC<ArcCursorProps> = ({ RADIUS, THETA }) => {
    const CENTER = { x: RADIUS, y: RADIUS };

    const { gestureHandler, translation, state } = usePanGestureHandler();
    
    // Convert X,Y canvas coordinates to Polar coordinates to get the angle value (theta)
    const CANVAS_X = withOffset(translation.x, state);
    const CANVAS_Y = withOffset(translation.y, state);
    const polar = canvas2Polar({
        x: CANVAS_X,
        y: CANVAS_Y
    }, CENTER);

    // Convert the angle value (theta) and the radius, into X,Y values representing the canvas coordinates system
    const { x: translateX, y: translateY } = polar2Canvas({
        theta: polar.theta,
        radius: RADIUS
    }, CENTER);

    return (
        <PanGestureHandler {...gestureHandler}>
            <Animated.View 
                style={{
                    ...StyleSheet.absoluteFillObject,
                    backgroundColor: "dodgerblue",
                    width: 40,
                    height: 40,
                    borderRadius: 20,
                    elevation: 2,
                    borderColor: "white",
                    borderWidth: 5,
                    transform: [ { translateX }]
                }}
            />
        </PanGestureHandler>
    );
};

export default ArcCursor;
```

Is there some bug with the `polar2canvas` method in the latest version or I'm not using it properly (since I was following your SVG Animations guide from June 2020, the one you posted on freecodecamp) ? 

Thanks in advance!

**Comment by wcandillon** [comment]
Is it because the new version of these function is designed for reanimated 2? Are you using the same version of redash than the one used in the github repos?

**Comment by pstevovski** [comment]
@wcandillon Oh I see, that is most likely the problem. My version of `react-native-reanimated` that is installed is `^1.13.2` , I just did `npm install --save react-native-reanimated` and I guess it doesn't installs Reanimated2 since its still marked as alpha? Also, the redash version, I just checked, is not the same as in the tutorial repo (there it is  `14.1.1` and I have `16.0.8` installed). Its clearer now :)
